yarn.lock

```diff
@@ -8352,6 +8352,15 @@ axios@1.7.4, axios@^1.6.0, axios@^1.6.2, axios@^1.7.4:
     form-data "^4.0.0"
     proxy-from-env "^1.1.0"
 
+axios@^1.7.9:
```

Comment:
@jeramysoucy this backport doesn't remove the old version of axios. Is there more we can do to get everything on `1.7.9`?

Reply:
Thanks for catching this, Larry. I missed that we had a resolution. We no longer need it so I am removing it.